tests/run/large_integer_T5290.py

```diff
@@ -0,0 +1,19 @@
+# cython: test_assert_c_code_has = __pyx_int_large_498767777577077770_xxx_709777877895072797
+# cython: test_assert_c_code_has = __pyx_int_large2_498767777577077770_xxx_709777877895072797
+
+"""
+>>> print(large_int1)
+498767777577077770556777587772706774776460369770778448231270737072737774657965287074870267\
+777936758056447717673778020572245677462377467777877973720727779047207424736977237577049760\
+567712761768711777573771777179179717700370131327713044788007971447377173777866449271749747\
+721187877247577877179557737776707777677777217729897579397557707379543777975071577397912797\
+751172283795307837046127757551976278271387413977777777470074377477377099877743777277619817\
+9733909687574772704167777795797966373977780709777877895072797
+>>> large_int2 - large_int1 == 10**485
+True
```

Comment:
If that test starts failing at some point, we'd better have some diagnostic output.
```suggestion
>>> large_int2 - large_int1 == 10**485  or  large_int2
True
```